syntaxes/graphql.json

```diff
@@ -342,7 +342,14 @@
       }
     },
     "graphql-directive": {
-      "begin": "\\s*((@)\\s*([_A-Za-z][_0-9A-Za-z]*))",
+      "contentName": "meta.embedded.block.directive.graphql",
+      "begin": "\\s*(?=@)",
+      "end": "(?=.)",
+      "applyEndPatternLast": 1,
+      "patterns": [{ "include": "#graphql-directive-body" }]
+    },
+    "graphql-directive-body": {
+      "begin": "((@)\\s*([_A-Za-z][_0-9A-Za-z]*))",
```

Comment:
I managed to get this down to a minimum amount of changes, but unfortunately not down to none.